deploy/crds/compliance.openshift.io_v1alpha1_rule_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: compliance.openshift.io/v1alpha1
```

Comment:
Can we remove this CR? It's not like it's an example CR that people are supposed to create.